src/commands/General/stats.js

```diff
@@ -1,45 +1,50 @@
-const process = require('process');
-const prettyMilliseconds = require('pretty-ms');
 const { EmbedBuilder, version } = require('discord.js');
-const { cpus } = require('os');
-const user = require('../../models/user.js');
+const prettyMilliseconds = require('pretty-ms');
+const { uptime, cpus } = require('os');
 
 module.exports = {
-  category: 'General',
-  clientPermissions: ['EmbedLinks'],
-  data: {
-    name: 'stats',
-    description: 'Shows the statistics of the bot.',
-  },
-  async run(client, interaction) {
-    const load = cpus()
-      .map((core) => {
-        let { user, nice, irq, sys, idle } = core.times;
-        return (
-          ((((user + nice + sys + irq) / idle) * 10000) / 100).toFixed(2) + '%'
-        );
-      })
-      .join(' | ');
+run: (config, interaction) => {
+  const {client} = interaction;
+  const { heapUsed, heapTotal } = process.memoryUsage();
 
-    const uptime = prettyMilliseconds(client.uptime, {
+  const statistics = {
+    'Discord.js': 'v' + version,
+    'Node.js': process.version,
+    'Servers': client.guilds.cache.size,
+    'Users': client.guilds.cache.reduce((acc, curr) => acc + (curr.memberCount), 0),
+    'Channels': client.channels.cache.size.toLco,
+    'CPU Load': cpus()
+      .map(c => {
+        let { user, nice, irq, sys, idle } = c.times;
```

Comment:
Codacy has a fix for the issue: ['user' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/Shadeslayer10/PokeRealm/pullRequest?prid=9275083)
```suggestion
        const { user, nice, irq, sys, idle } = c.times;
```